src/foam/nanos/crunch/OptionalBoolean.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @license
+ * Copyright 2022 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+ foam.ENUM({
+    package: 'foam.nanos.crunch',
```

Comment:
This has nothing to do with crunch, we need a better package. 

```suggestion
    package: 'foam.core',
```